lib/src/model/never.dart

```diff
@@ -2,18 +2,23 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-// ignore_for_file: analyzer_use_new_elements
-
 import 'package:analyzer/dart/element/element.dart';
+import 'package:analyzer/dart/element/element2.dart';
+// ignore: implementation_imports
```

Comment:
We'll need to remember to search for imports of these extensions before we declare the package fully converted.

Alternatively, you could remove `element` and use `.element2.asElement` in any unconverted files. That might be less error prone.

Reply:
Got it. Will look into this in a follow up CL.